README.md

```diff
@@ -1,10 +1,70 @@
 # disblob
+A stylesheet(css) generator for use with Discord.
+***
+***All original code is done by the insane but genius [diamondburned](https://github.com/diamondburned/)***
+> [Master repo](https://github.com/diamondburned/disblob)
 
-A stylesheet generator for use with Discord.
+***This fork is an attempt to refine the `README` and make it easier for others to setup!***
+***
+# Status
+|   |Status |	|
+|--|--|--|
+|   |Edits: |	|
+|  | `README` Only |  |
 
+|   |Fork|  |
+|--|--|--|
+|  | Fork Only |  |
+|  |  (PR [Open](https://github.com/diamondburned/disblob/pull/1))  |  |
+
+|  | Testing |  |
+|--|--|--|
+|  | 		-Client + Firefox-		 |  |
+|  |-  (Tests have been done with the client and Firefox)  |  
+***
+ - [x] Client (BD(BetterBiscord)~(100% Working!))
+ - [ ] Client (Other css injectors)
+ - [ ] Browsers (Chrome, etc. (Use [stylus](https://github.com/openstyles/stylus) to inject css)
+ - [x] Firefox Browser (Testing done with stylish(DO NOT USE(Considered spyware)) 
+***
+***
+|  | Contributors:|  |
+|--|--|--|
+|  | 		2 ([diamondburned](https://github.com/diamondburned/) & [ThatGeekyWeeb](https://github.com/ThatGeekyWeeb))
+
+***
+## Dependencies
+  1. (All Blobmoji deps(https://github.com/C1710/blobmoji/wiki/Build-instructions))
+  2. `go`
+  3. `zopfli`
+  4. 1+ GiB of storage space
+ 
 ## Building
 
 ```sh
+git clone https://github.com/diamondburned/disblob.git
+cd disblob
+git submodule update --init
+cd blobmoji
+make -j$(nproc)
+cd ..
+mv blobmoji/build/renamed_flags/* blobmoji/svg/
+mv blobmoji/build/resized_flags/ blobmoji/
+
 # This will generate the stylesheet.
-go run . -defpath definitions.css -datpath data.css
+go run . -defpath definitions.css -datpath data.css > <css file>
+```
+## Releases
+My github pages link can be used to `@import` the css
+```css
+@import url("https://thatgeekyweeb.github.io/src/import.data.css");
 ```
+
+Additionally there are releases [here!](https://github.com/ThatGeekyWeeb/disblob/releases)
+
+
+# Contributing
+Any Pull Requests, Opened Issues, and forks are  Greatly welcomed!\
+You help make disblob better by Contributing!
+
+> Written with [StackEdit](https://stackedit.io/)
```

Comment:
Remove.

Reply:
Alright, I had it down to compile blobmoji as it had trouble finding the flag emojis as they aren't in `blobmoji/svg` but rather `blobmoji/build/rename_flag/` after blobmoji is compiled, but, they are still optional, but the blobmoji flags will not be generated in the css unless blobmoji is compiled , and the flags are than moved, 

```bash
mv blobmoji/build/renamed_flags/* blobmoji/svg/
mv blobmoji/build/resized_flags/ blobmoji/
```
Although, still not required, so so I'll label them as optional